native/effektio/api.rsh

```diff
@@ -16,6 +16,12 @@ fn register_with_registration_token(basepath: string, username: string, password
 /// generate news mock items
 fn gen_mock_news() -> Vec<News>;
 
+/// Representing a time frame
+object Duration {}
+
+fn duration_from_secs(secs: u64) -> Duration;
+
+
 /// Representing a color
 object Color {
```

Comment:
I saw that Color class has been implemented in rust which makes Flutter confuse as to whom it is referencing when mapping the object to Flutter class ```ToDoList```. ```Color``` is also present in the Flutter core ```painting.dart```.

Reply:
nothing new in this one, but we can rename it here if you like. `EfkColor` or `MatrixColor` what do you prefer?